daemon/mgr/container.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/alibaba/pouch/pkg/errtypes"
 	"github.com/alibaba/pouch/pkg/meta"
 	mountutils "github.com/alibaba/pouch/pkg/mount"
+	ns "github.com/alibaba/pouch/pkg/namespace"
```

Comment:
package named `namespace` is not suitable, you just move the container's network functions into the common package, in my opinion, named `net` is better， but I don't agree move then into pkg.

Reply:
network function is one part of namespace, and some function are repeated, so make them as a pkg, wdyt?